Makefile

```diff
@@ -3,6 +3,9 @@ all:	gets application
 linux:
 	GOOS=linux GOARCH=amd64 make application
 
+linux-win:
+	set GOOS=linux && set GOARCH=amd64 && make application
+
```

Comment:
For consistency, can you please use bash syntax, avoiding `set` and `&&`?

Reply:
ok - then I remove it again